src/ui/components/SecondaryToolbox/react-devtools/ReplayWall.ts

```diff
@@ -168,7 +168,10 @@ export class ReplayWall implements Wall {
           // Get the first node ID we found for this fiber ID, if available.
           const [nodeId] = fiberIdsToNodeIds.get(fiberId) ?? [];
           if (!nodeId) {
-            sendTelemetryEvent("reactdevtools.node_not_found", payload);
+            // skip on hitting this again for the same fiberId (maybe combo it with pauseId?)
```

Comment:
Mixed feelings here, but I think we should either remove the "maybe" comment or make it more concrete.

Replay's persistent Fiber ids are supposed to be the same for the entire recording, so that would be a reason to only key on the id as you're doing. However, if we can't find one at a particular execution point, that could indicate something unexpected which would be a reason to include the pause id in the data at least (even if not distinct on that id)

Reply:
Sorry about that, this was a comment I added when I noticed the problem last week and it was by no means meant to be committed 🤦‍♂️

Reply:
No worries :)